File: LevelAndroidSDK/sdk/src/main/java/com/theshopatvsp/levelandroidsdk/ble/model/response/LockPacket.java
package com.theshopatvsp.levelandroidsdk.ble.model.response;

/**
 * Created by andrco on 7/15/16.
 */
public class LockPacket extends DataPacket {
    private int lock;

    public LockPacket(int lock) {
        super();
        this.lock = lock;
    }

    public LockPacket(byte bytes[]) {
        this.lock = bytes[2];
    }

    public byte[] getPacket() {

        return null;
    }

    public int getLock() {
        return lock;
    }
}

File: LevelIOSSdkExample/LevelIOSSdkExample/LevelIOSSdk/model/LockPacket.swift
//
//  LockPacket.swift
//  LevelIOSSDK
//
//  Created by Andrew Cook on 7/5/16.
//  Copyright © 2016 TheShop. All rights reserved.
//

import Foundation

class LockPacket: DataPacket {
    var lock: Int
    
    override init() {
        self.lock = 0
        super.init()
    }
    
    convenience init(code: Int) {
        self.init()
        
        self.lock = code
    }
    
    convenience init(bytes: [UInt8]) {
        self.init(code: Int(bytes[2]))
    }
}